### ISSUE TITLE
Move from master to gh-pages branch?

### ISSUE THREAD

**Issue by jbranchaud** [jbranchaud/splitting-atoms#45]
I am thinking of moving everything from the `master` branch (ditching it) to the `gh-pages` branch and then developing there as a Jekyll/GitHub Pages built site. I am now realizing that the master branch doesn't have much use if it is all markdown content that is intended to be rendered by GitHub pages. Thoughts on this? Reasons to not ditch the `master` branch?

I think this transition would also be a good opportunity to deal with #44.

**Comment by afj176** [comment]
:+1:

**Comment by ZucchiniZe** [comment]
Wow, just about to ask you about this. I was thinking of keeping the `master` branch and using [this](https://github.com/mnapoli/Couscous) to push to a `gh-pages` branch so you could still view the `master` branch.

**Comment by jbranchaud** [comment]
@ZucchiniZe We can just as easily set `gh-pages` as the main branch for this repo and remove master, this is done, for example, with [backbone fundamentals](https://github.com/addyosmani/backbone-fundamentals).

**Comment by ZucchiniZe** [comment]
True, But with Couscous the writers can still submit in markdown instead of having to edit 1 giant html file.

**Comment by rgbkrk** [comment]
@ZucchiniZe Can't we still just submit in markdown with `gh-pages`? My github.io blog certainly uses markdown explicitly.

I am wondering if the build should be separate so we have an ePub as well. Read the docs manages that well too.

**Comment by jbranchaud** [comment]
GitHub Pages fully supports writing content in markdown. It uses [Jekyll](https://github.com/jekyll/jekyll) in the background to generate the static sites.

@rgbkrk At some point I would like to setup a build process for ePub, PDF, etc., but for now while this thing continues to take shape and evolve, I am leaning towards keeping things simple. As we strike a balance between structured content and regular blog posts, it will become more clear how we will move in the direction of other formats. Thoughts?

**Comment by rgbkrk** [comment]
@jbranchaud :+1: charging forward with markdown and gh-pages!

**Comment by ZucchiniZe** [comment]
Oh wow, just **now** I am realizing how stupid I was.

**Comment by jbranchaud** [comment]
I have some preliminary stuff setup on `gh-pages` now. Still needs a lot of work and probably a complete design overhaul at some point, but at least it is a step towards making all the content more accessible.
